src/schema/v2/order/types/DisplayTexts.ts

```diff
@@ -33,6 +33,21 @@ const DisplayTextsType = new GraphQLObjectType<any, ResolverContext>({
         return formatMarkdownValue(message, format)
       },
     },
+    // TODO: deprecated - remove after deploy
```

Comment:
I think there is a GraphQL way to [deprecate](https://github.com/artsy/metaphysics/blob/254b7dae27a1845391bab2fab04e38f84c62cb67/src/schema/v2/conversation/index.ts#L414) fields. Hopefully quick change and regenerate the schema.